### PR TITLE
Code snippet for Longest Common Subsequence

### LINKED ISSUE #3: Longest Common Subsequence
# Description

Write a program to find the longest common subsequence of a string.

## Language

- [x] C++: @SrijitaSarkar99 
- [x] Java: @Bhaswati-Saha
- [x] Python: @ShivamPorwal02 

## Type of issue

- [x] Feature
- [ ] Bug

### PR DESCRIPTION
**Pull request by ShivamPorwal02**
# Code snippet for Longest Common Subsequence

Fixes #3 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generates no new warnings

### CODE REVIEW

**Review by sakshamarora1 — CHANGES_REQUESTED** [review]
Thank you for opening a Pull Request @ShivamPorwal02 :D
Just a few things

File: snippets/python/Longest_common_subsequence.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Least Common Subsequence
+
+## Code
+
+```python
+# Input strings
```
Comment (sakshamarora1) [inline snippets/python/Longest_common_subsequence.md]: We should only keep important comments. Please see to that. :)

File: snippets/python/Longest_common_subsequence.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Enter String2: CBDA
+
+LCS: CB
+```
+## Contributed By
+
+| Name | GitHub Username | Institute |
+| --- | --- | --- |
+| Shivam Porwal | ShivamPorwal02 | Maharaja Surajmal Intitute of Technology |
```
Comment (sakshamarora1) [inline snippets/python/Longest_common_subsequence.md]:
```suggestion
| Shivam Porwal | ShivamPorwal02 | Maharaja Surajmal Institute of Technology |
```

**Review by sakshamarora1 — CHANGES_REQUESTED** [review]
Thank you for opening a Pull Request @ShivamPorwal02 :D
Just a few things
Also please edit the PR description accordingly. Thanks

File: snippets/python/Longest_common_subsequence.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Least Common Subsequence
```
Comment (kaustubhgupta) [inline snippets/python/Longest_common_subsequence.md]: *longest

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by ShivamPorwal02** [comment]
@kaustubhgupta @sakshamarora1 I have made the requested changes, please review.

**Review by kaustubhgupta — APPROVED** [review]

**Review by sakshamarora1 — APPROVED** [review]

**Pull request merged**